### ISSUE TITLE
Shorter, more memorable URL?

### ISSUE THREAD

**Issue by tabatkins** [w3c/PWETF#145]
Currently, to access the CEPC, one must remember that it is namespaced under the `/Consortium/` folder, and that it is named "cepc" rather than the more standard "coc" acronym.

Any chance we could install a redirect on `w3.org/coc` (or perhaps `/conduct` or `/code-of-conduct`?) that directs to this document, so there's a short, immediately memorizable URL to this important document?

**Comment by TzviyaSiegman** [comment]
Thanks @tabatkins What do you think of https://www.w3.org/Consortium/cepc/?

**Comment by TzviyaSiegman** [comment]
ah, i didn't read carefully. @swickr - I like Tab's suggestion. Can we look into this?

**Comment by swickr** [comment]
Short URIs can be overly language/culture specific.  hrefs are inexpensive. 

I would prefer a new entry on "About W3C" -> "quick links" with the label "Code of Conduct ".  (I.e. duplicating the entry that is in "Participate".)

**Comment by tabatkins** [comment]
> Short URIs can be overly language/culture specific

That might be a reasonable general principle to work with, but I don't see how it applies here. The url is currently `w3.org/Consortium/cepc`. I'm asking for a redirect on `w3.org/coc`. Both seem, at worst, identically language/culture specific. At best, "CoC" is a more widely recognized worldwide acronym for code-of-conduct than "cepc", which to my experience appears to be a freshly-minted acronym with no existing cachet.

> hrefs are inexpensive.

But remembering them isn't. I'm asking for a short url not because typing is hard, but because my memory is bad, and I'll want to be reference this URL at random moments when responding to what is likely already a stressful thread; the easier it is to recall the better. I don't want to have to interrupt my train of thought to go search for this document because I can't remember its URL, and definitely can't guess it.

> I would prefer a new entry on "About W3C" -> "quick links" with the label "Code of Conduct ". (I.e. duplicating the entry that is in "Participate".)

This also sounds excellent, yes please.

**Comment by hober** [comment]
I agree with @tabatkins that it would be a good thing to mint a more memorable URL for the CEPC, which would redirect to whatever the canonical URL is.

FWIW, the WHATWG's [Code of Conduct](https://whatwg.org/code-of-conduct) is at the very memorable URL `https://whatwg.org/code-of-conduct`.
